test/gp/lti_sde.jl

```diff
@@ -26,7 +26,7 @@ println("lti_sde:")
             # (name="static storage Float32", val=SArrayStorage(Float32)),
         )
 
-        kernels = [Matern12Kernel(), Matern32Kernel(), Matern52Kernel()]
+        kernels = [Matern12Kernel(), Matern32Kernel(), Matern52Kernel(), ConstantKernel()]
```

Comment:
```suggestion
        kernels = [Matern12Kernel(), Matern32Kernel(), Matern52Kernel(), ConstantKernel(c=1.5)]
```
Since there's a parameter in `ConstantKernel`.